Cargo.toml

```diff
@@ -31,3 +31,8 @@ critical-section = "1.1"
 
 [build-dependencies]
 riscv-target = "0.1.2"
+
+[features]
+default = ["rt", "critical-section"]
+rt = ["bl602-pac/rt"]
+critical-section = ["bl602-pac/critical-section", "riscv/critical-section-single-hart"]
```

Comment:
Cargo is upset with the name of this feature.
The most common name used for this is: `critical-section-impl` if you weren't sure what to name it.

Reply:
I'll also remove the `rt` feature, looking at it again I don't immediately see a reason someone would want to enable `rt` in the pac.

Reply:
You need it to generate an interrupt vector table. Without it, I don't think the interrupt example will work correctly.

Reply:
Interrupt example is still working with this PR. Maybe I'm wrong about this?

Reply:
I think because `riscv-rt` is a dev dependency.
Do you think it's useful to add the `rt` feature again?

Reply:
It wasn't there before, so lets leave it for a separate PR once we've worked out if/where it's required.